io/net/infiniband/ib_latency_perf.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Prudhvi Miryala<mprudhvi@linux.vnet.ibm.com>
+# Latency Performance test for infiniband adaptors
+# ib_send_lat test
+# ib_write_lat test
+# ib_read_lat test
+# ib_atomic_lat test
+
+
+import time
+import netifaces
+from avocado import Test
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import process
+from avocado.utils import distro
+
+
+class Latency_Perf(Test):
+    '''
+    Infiniband adaptors latency performance tests using four tools
+    tools are ib_send_lat,ib_write_lat,ib_read_lat,ib_atomic_lat
+    '''
+    def setUp(self):
+        '''
+        check the availability of perftest package installed
+        perftest package should be installed
+        '''
+        sm = SoftwareManager()
+        detect_distro = distro.detect()
+        depends = ["openssh-clients"]
+        if detect_distro.name == "Ubuntu" or detect_distro.name == "redhat":
+            depends.append("perftest")
```

Comment:
sorry i miss this point in our earlier discussion  , i think perftest rpm also part of fedora and centos  as well SLES or open suse , can this test not going to support these distribution please help me on this